app/models/concerns/generic_phrase_concern.rb

```diff
@@ -2,12 +2,14 @@ module GenericPhraseConcern
   extend ActiveSupport::Concern
 
   class_methods do
```

Comment:
As far as I can tell, this module only gets included once. Have I missed something? If not, it'd be nice to move it to `GenericPhrase`

Reply:
It does indeed only get called once, but in the spirit of making the model purely the model without any ancillary functions I decided to move this out into a separate module. Happy to merge back in if you think that's better, but ideally I'd like to keep this separate.

Reply:
Actually, having already created one private method in `generic_phrase` I'm happy to move this back out into that model for consistency.

Reply:
Moved back into the model 👍